changelog.md

```diff
@@ -5,3 +5,13 @@ Dexer is used to index the files in a directory, this helps to search a given wo
 
 Dexer use case is dgplug logs to search a given word in the logs and see where it has been talked about.
```

Comment:
```suggestion
 Dexer is being used to search a given word in the dgplug logs and see where it has been discussed about.
```
Not a grammar nazi, but it may sound better, :wink: